berkeley-mobile/StudyPact/Profile/ProfileViewController.swift

```diff
@@ -418,11 +466,74 @@ extension ProfileViewController {
     }
     
     @objc private func cancelButtonPressed(sender: UIButton) {
-        // TODO: call GetUser again and fill in fields
+        fullNameField.setDefault()
+        phoneTextField.setDefault()
+        facebookTextField.textField.setDefault()
+        
+        // Get user info from backend
+        StudyPact.shared.getUser { (data) in
+            if data != nil {
+                self.emailTextField.textField.text = data!["user_email"] as? String
+                
+                let info = data!["info"] as! [String : String]
+                
+                self.fullNameField.text = info["name"]
+                if let phone = info["phone"] {
+                    self.phoneTextField.text = phone
+                }
+                if let fb = info["facebook"] {
+                    self.facebookTextField.textField.text = fb
+                }
```

Comment:
if previously the user didn't have a facebook/phone on backend this doesn't work. if it's nil on backend we should set the text field to be empty ""